genome/bin_statistic_ext.py

```
@@ -15,16 +15,19 @@
 
 import pandas as pd
 
-from .bin_statistic import contig2bin
+from .bin_statistic import contig2bin, Binput
```

Ruff (0.11.9)

`.bin_statistic.contig2bin` imported but unused

Remove unused import: `.bin_statistic.contig2bin`

(F401)